### PR TITLE
Make stripping of metadata optional

### PR DESCRIPTION
**Pull request by rayshan**
It's useful to preserve metadata sometimes, such as for a photographer wishing to preserve copyright info.

Fixture was modified only to add a piece of EXIF metadata for testing.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,6 +73,10 @@ module.exports = function (opts) {
 			args.push('-S', opts.subsample);
 		}
 
+		if (opts.strip) {
```
Comment (shinnn) [inline index.js]:
```js
if (opts.strip !== false) {
```

`strip` option should be enabled by default.

**Comment by shinnn** [comment]
https://github.com/imagemin/imagemin-jpeg-recompress#api needs update.

**Comment by rayshan** [comment]
All done, let me know if a squash is preferred.

**Comment by sindresorhus** [comment]
LGTM

@rayshan Can you fix the merge conflict?

**Comment by rayshan** [comment]
I'm not able to run tests on a new master checkout. Need to spend some time to figure it out...

```
❯ npm test

> imagemin-jpeg-recompress@5.0.0 test /Users/rayshan/=Projects=/imagemin-jpeg-recompress
> xo && ava

WIP - Simple test runner

Usage
  $ ava <file>

Example
  $ ava test.js

  ✔ 0 tests passed
```

**Comment by sindresorhus** [comment]
@rayshan Remove your `node_modules` folder and reinstall.

**Comment by leachiM2k** [comment]
Hello,
I would like to vote +1 for this pull request, cause we really need this feature.

**Comment by kevva** [comment]
@leachiM2k, it'll be merged once the merge conflicts are resolved.

**Comment by rayshan** [comment]
Sorry for the delay, I'll be working on this today.

**Pull request closed without merging**